pythonforandroid/recipes/av/__init__.py

```diff
@@ -0,0 +1,25 @@
+from pythonforandroid.toolchain import Recipe
+from pythonforandroid.recipe import CythonRecipe
+
+
+class PyAVRecipe(CythonRecipe):
+
+    name = "av"
+    version = "v10.0.0"
+    url = "https://github.com/PyAV-Org/PyAV/archive/v10.0.0.zip"
```

Comment:
The `{version}` should be used in the URL so it can be overridden dynamically.
```suggestion
    version = "10.0.0"
    url = "https://github.com/PyAV-Org/PyAV/archive/v{version}.zip"
```